test/README.md

```diff
@@ -90,3 +90,13 @@ The program will wait until you launch the Debug mode from VS Code (next step).
 Launch debug session by selecting Run/Start Debugging from top menu.
 
 Further information [here](https://github.com/Microsoft/vscode-go/wiki/Debugging-Go-code-using-VS-Code).
+
+## Miscellaneous
```

Comment:
I would not create a new big section. I would let it in `Development` section or move to the `Run tests` section

Reply:
Changed.